### DISCUSSION TITLE
drop-down menu for changing camera does not work but there is a front camera option

### DISCUSSION THREAD

**Discussion by RRDMELO** [community/community#141705] [GitHub Education]
### Hi everyone,

Optionally, use the camera drop-down menu to change the camera you want to use if you have multiple cameras available.
It's not working, only the front camera appears, it's impossible to take a levigel photo without being able to see what I'm photographing. 
Would there be another way to send the documents for verification to the educational institute I attend?

**Comment by itzzexcel** [comment]
I've searched for this since August of this year, looks like the bug is there since more than 1 year ago.
